models/naughty-words.json

```diff
@@ -0,0 +1,444 @@
+[
```

Comment:
We shouldn't include this wordlist in the source control. It's one thing to provide a facility for users to specify words that they don't want used in their community and another to broadly scope words that are completely valid in circumstances.

I had reviewed with the team an idea like this before when my first pass against the wordlist started, and I don't think that including this is a good idea.